downstream/modules/platform/ref-eda-system-requirements.adoc

```diff
@@ -0,0 +1,16 @@
+[id="event-driven-ansible-system-requirements]
```

Comment:
You need to add close quotes at the end of your id definition: [id="event-driven-ansible-system-requirements**"**]. This should fix the build issue you're seeing.

Reply:
Will do! Thanks for finding this "not-so-obvious" error!